src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Orchestrators/IncentivePaymentOrchestrator.cs

```diff
@@ -66,28 +66,18 @@ public async Task RunOrchestrator([OrchestrationTrigger] IDurableOrchestrationCo
                 _logger.LogInformation("[IncentivePaymentOrchestrator] Calculated payments for collection period {collectionPeriod} have been rejected", collectionPeriod);
                 return;
             }
-
-            context.SetCustomStatus("SendingPayments");
+            
+            context.SetCustomStatus("SendingClawbacksAndPayments");
             _logger.LogInformation("[IncentivePaymentOrchestrator] Calculated payments for collection period {collectionPeriod} have been approved", collectionPeriod);
 
             var sendPaymentTasks = new List<Task>();
             foreach (var legalEntity in payableLegalEntities)
             {
-                var sendPaymentTask = context.CallActivityAsync(nameof(SendPaymentRequestsForAccountLegalEntity), new AccountLegalEntityCollectionPeriod { AccountId = legalEntity.AccountId, AccountLegalEntityId = legalEntity.AccountLegalEntityId, CollectionPeriod = collectionPeriod });
+                var sendPaymentTask = context.CallSubOrchestratorAsync(nameof(SendPaymentsForAccountLegalEntityOrchestrator), new AccountLegalEntityCollectionPeriod { AccountId = legalEntity.AccountId, AccountLegalEntityId = legalEntity.AccountLegalEntityId, CollectionPeriod = collectionPeriod });
                 sendPaymentTasks.Add(sendPaymentTask);
             }
```

Comment:
There is potential for this to miss people as the collection of legal entities with a clawback won't necessarily be the same as the collection with a payment.  You'll need to get a distinct list from the two collections.

Reply:
Updated